### PR TITLE
fixes #11699 - support for global status

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by stbenjam** [comment]
Needs https://github.com/theforeman/foreman/pull/2725 to pass tests

File: app/helpers/concerns/foreman_salt/layout_helper_extensions.rb [hunk 1]
```diff
@@ -1,20 +0,0 @@
-module ForemanSalt
-  module LayoutHelperExtensions
-    extend ActiveSupport::Concern
-
-    included do
-      alias_method_chain :authorized_associations_permission_name, :salt
```
Comment (stbenjam) [inline app/helpers/concerns/foreman_salt/layout_helper_extensions.rb]: This method was added during 1.10-develop and then replaced by the AssociationAuthorizer, and I'm really not sure how to extend that cleanly.  *sigh*

Comment (lzap) [inline app/helpers/concerns/foreman_salt/layout_helper_extensions.rb]: Damn I am just trying out salt for my LinuxDAYS 15 talk for the first time and it does not work.

**Comment by lzap** [comment]
Oh great with this patch, 1.10 stable branch now boots up! Kudos. [test]

**Comment by mmoll** [comment]
seems to make things working for me again, what's up with the tests? :)

**Comment by stbenjam** [comment]
Test: http://ci.theforeman.org/view/Plugins/job/test_plugin_pull_request/1531/, to see if anything else is broken.

At least one will fail until https://github.com/theforeman/foreman/pull/2803 is accepted, or I monkeypatch some stuff.

File: app/helpers/concerns/foreman_salt/layout_helper_extensions.rb (hunk 1 above)
Comment (stbenjam) [inline app/helpers/concerns/foreman_salt/layout_helper_extensions.rb]: Yea, foreman_salt touches a lot of stuff and tends to break between major releases.  It'll be working in time for the RC, hopefully.  

3.x should work on foreman 1.9 stable.

**Comment by stbenjam** [comment]
Green http://ci.theforeman.org/view/Plugins/job/test_plugin_pull_request/1539/

**Pull request merged**